lstchain/reco/dl0_to_dl1.py

```diff
@@ -166,63 +183,84 @@ def r0_to_dl1(input_filename=get_dataset_path('gamma_test_large.simtel.gz'), out
 
     cal = load_calibrator_from_config(config)
 
-    dl1_container = DL1ParametersContainer()
+    if not is_simu:
+        #TODO : add calibration config in config file, read it and pass it here
+        charge_config = Config()
+        r0_r1_calibrator = LSTR0Corrections(pedestal_path=pedestal_path,
+                                            r1_sample_start=2,  # numbers in config ?
+                                            r1_sample_end=38,
```

Comment:
If the config file includes a `LSTR0Corrections(...)` configuration they shouldn't be re-declared. 
Otherwise there should be a camera_config_file (for example), or just declare them by hand.

Reply:
Not sure to completely understand what you mean.
To clarify:
current `config` in lstchain in a simple dictionnary. However `charge_config` must be a `Traitlet Config`. Here empty because #TODO.